VideoLocker/src/main/java/org/edx/mobile/model/BaseComponent.java

```diff
@@ -0,0 +1,40 @@
+package org.edx.mobile.model;
+
+/**
+ * Created by hanning on 4/29/15.
+ */
+public class BaseComponent implements IComponent {
```

Comment:
Do you think component is too vague a name? Maybe it should be "CourseComponent" or something like that? I'm calling it a CourseBlock